core/templates/components/shared-component.module.ts

```diff
@@ -40,6 +40,8 @@ import { ExplorationEmbedButtonModalComponent } from
 import { SocialButtonsComponent } from
   'components/button-directives/social-buttons.component';
 import { NgbModalModule } from '@ng-bootstrap/ng-bootstrap';
+import { TopicSummaryTileComponent } from
+  './summary-tile/topic-summary-tile.component';
```

Comment:
Hey @orthodoxparadox I don't think we use relative imports?

Reply:
The old imports were relative, so I just followed the pattern. Although my earlier PR (Social Buttons) did have absolute imports. Should I convert all the imports to absolute?